mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -1033,6 +1033,16 @@
 									"        pm.environment.set(\"custom-package-id-created-in-post\", response.data.id);",
 									"    }",
 									"    ",
+									"//Check that tagList is not empty",
+									"pm.test('TagList is not empty', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+									"});",
+									"    ",
+									"//Check that tagList is not empty",
+									"pm.test('Creating tagList', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+									"});",
+									"    ",
```

Comment:
Also makes sense to have checks on tags.tagList after we have confirmed that keys includes tags (similar to where the check of other content is done)